src/main/java/org/alliancegenome/curation_api/services/OrganizationService.java

```diff
@@ -1,21 +1,28 @@
 package org.alliancegenome.curation_api.services;
 
-import io.quarkus.logging.Log;
-import jakarta.annotation.PostConstruct;
-import jakarta.enterprise.context.RequestScoped;
-import jakarta.inject.Inject;
+import java.util.Date;
+import java.util.HashMap;
+
+import org.alliancegenome.curation_api.auth.AuthenticatedUser;
 import org.alliancegenome.curation_api.dao.OrganizationDAO;
+import org.alliancegenome.curation_api.model.entities.AllianceMember;
 import org.alliancegenome.curation_api.model.entities.Organization;
+import org.alliancegenome.curation_api.model.entities.Person;
 import org.alliancegenome.curation_api.response.ObjectResponse;
 import org.alliancegenome.curation_api.response.SearchResponse;
 import org.alliancegenome.curation_api.services.base.BaseEntityCrudService;
 
-import java.util.Date;
-import java.util.HashMap;
+import io.quarkus.logging.Log;
+import jakarta.annotation.PostConstruct;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
 
 @RequestScoped
 public class OrganizationService extends BaseEntityCrudService<Organization, OrganizationDAO> {
 
+	@Inject
+	@AuthenticatedUser
+	protected Person authenticatedPerson;
```

Comment:
Is this field not injected in the super class `BaseEntityCrudService`? I would think it should be done at the super class level.

Reply:
Good spot, injection removed